src/CMakeLists.txt

```diff
@@ -9,7 +9,17 @@ if (USE_CUDA)
   file(GLOB_RECURSE CUDA_SOURCES *.cu *.cuh)
   target_sources(objxgboost PRIVATE ${CUDA_SOURCES})
   target_compile_definitions(objxgboost PRIVATE -DXGBOOST_USE_CUDA=1)
-  target_include_directories(objxgboost PRIVATE ${xgboost_SOURCE_DIR}/gputreeshap)
+
+  # GPUTreeSHAP
+  include(FetchContent)
+  FetchContent_Declare(
+    GPUTreeSHAP
+    GIT_REPOSITORY https://github.com/hcho3/gputreeshap
```

Comment:
Need to switch to rapidsai/gputreeshap after https://github.com/rapidsai/gputreeshap/pull/1 is merged.